src/components/menu/menu.tsx

```diff
@@ -39,6 +30,12 @@ export class Menu {
     @Prop({ reflectToAttr: true })
     public disabled = false;
 
+    /**
+     * Decides if the menu should open right or left. Defaults to right.
+     */
+    @Prop({ reflectToAttr: true })
+    public openDirection: OpenDirection = 'right';
```

Comment:
Defaulting to the right is a good choice I think, but it does kind of make this change breaking…

![image](https://user-images.githubusercontent.com/154725/78022437-36619280-7355-11ea-9e81-ddc5b06090d9.png)

I'm fixing the above in Lundalogik/lime-webclient#1247

Reply:
There are more issues with the menu... opened an issue yesterday in the regular board:
https://github.com/Lundalogik/crm-feature/issues/1008

Reply:
Ah, ok. Good that they've been caught 🙂